samples/create-and-edit-geometries/src/main/java/com/esri/arcgismaps/sample/createandeditgeometries/screens/ButtonMenu.kt

```diff
@@ -53,46 +66,90 @@ fun ButtonMenu(
             ) {
                 DropdownMenuItem(
                     text = { Text("Point") },
                     onClick = {
                         onStartEditingButtonClick(GeometryType.Point)
-                        expanded = false
+                        drawMenuExpanded = false
                     }
                 )
                 DropdownMenuItem(
                     text = { Text("Multipoint") },
                     onClick = {
                         onStartEditingButtonClick(GeometryType.Multipoint)
-                        expanded = false
+                        drawMenuExpanded = false
                     }
                 )
                 DropdownMenuItem(
                     text = { Text("Polyline") },
                     onClick = {
                         onStartEditingButtonClick(GeometryType.Polyline)
-                        expanded = false
+                        drawMenuExpanded = false
                     }
                 )
                 DropdownMenuItem(
                     text = { Text("Polygon") },
                     onClick = {
                         onStartEditingButtonClick(GeometryType.Polygon)
-                        expanded = false
+                        drawMenuExpanded = false
+                    }
+                )
+            }
+        }
+        IconButton(
+            enabled = canGeometryEditorUndo,
+            onClick = { onUndoButtonClick() }
+        ) {
+            Icon(imageVector = vector.vectorResource(R.drawable.undo_24), contentDescription = "Undo")
+        }
+        IconButton(
+            enabled = canGeometryEditorRedo,
+            onClick = { onRedoButtonClick() }
+        ) {
+            Icon(imageVector = vector.vectorResource(R.drawable.redo_24), contentDescription = "Redo")
+        }
+        Box(
+            modifier = Modifier
+        ) {
+            IconButton(
+                onClick = { deleteMenuExpanded = !deleteMenuExpanded }
+            ) {
+                Icon(imageVector = Icons.Filled.Delete, contentDescription = "Delete Geometry Menu")
+            }
+            DropdownMenu(
+                expanded = deleteMenuExpanded,
+                onDismissRequest = { deleteMenuExpanded = false }
+            ) {
+                DropdownMenuItem(
+                    text = { Text("Delete Selected Element") },
+                    enabled = canDeleteSelectedGeometry,
+                    onClick = {
+                        onDeleteSelectedElementButtonClick()
+                        deleteMenuExpanded = false
+                    }
+                )
+                DropdownMenuItem(
+                    enabled = !isGeometryEditorStarted,
+                    text = { Text("Delete All Geometries") },
+                    onClick = {
+                        onDeleteAllGeometriesButtonClick()
+                        deleteMenuExpanded = false
                     }
```

Comment:
There's never a time when both these options are available, which confused me at first. It's probably fine, but could there be a better solution? (EG 1 button, no menu, which changes behaviour depending on whether GE is started or not)